rholang/src/main/scala/coop/rchain/rholang/interpreter/accounting/Costs.scala

```diff
@@ -5,16 +5,20 @@ import com.google.protobuf.ByteString
 import coop.rchain.models.{PCost, Par, ProtoM, StacksafeMessage}
 
 //TODO(mateusz.gorski): Adjust the costs of operations
-final case class Cost(value: Long) extends AnyVal {
-  def *(base: Int): Cost   = Cost(value * base)
-  def +(other: Cost): Cost = Cost(value + other.value)
-  def -(other: Cost): Cost = Cost(value - other.value)
+final case class Cost(value: Long, operation: String) {
+  def *(base: Int): Cost   = Cost(value * base, s"$operation * $base")
```

Comment:
please add parentheses, especially after including the discarded 'operation'-s for + and -

Reply:
Where?